developers/weaviate/client-libraries/typescript/typescript-v3.md

```diff
@@ -399,87 +336,36 @@ Only Read operations powered by GraphQL.
-
-## TypeScript for JavaScript users
-
-TypeScript is a superset of JavaScript. There are, however, some differences that you should be aware of. This section offers some suggestions for JavaScript users who are new to TypeScript.
-
-### Run a TypeScript file
-
-To run a TypeScript file, first convert it to JavaScript. The `typescript` package from `npm` includes the `tsc` utility. Use `tsc` to convert (transpile) the TypeScript file. 
-
-Install the `typescript` package. Add the `-g` flag if you want the package to be available globally.
-
-```bash
-npm install typescript
-```
-You can then use this command to transpile the TypeScript file. 
-
-```bash
-tsc
-```
-
-`node` only allows the `import` statement in modules. To allow the `import` statement, add the following to your `package.json` file.
-
-```json
-{ 
-   "type": "module"
-}
-```
-
-### Example
+## What's Next
 
-To run this example, complete these steps. 
+Here are some resources to help you get started using the client.
 
-- Install the `typescript` package. 
-- Update the `tsconfig.json` and `package.json` files as described above.
-- Copy the sample code.
-- Save the code as `sample.ts` in the same directory as `tsconfig.json` and `package.json`.
-- Convert and run the code.
+### Recipes
 
-Use this code to create `sample.ts`.
+The [recipes repository](https://github.com/weaviate/recipes-ts) on Github has sample code for common use cases.
 
-<details>
-  <summary>Sample TypeScript code</summary>
+### Demo Applications
 
-```ts
-import weaviate, { WeaviateClient } from 'weaviate-client'
+There are demo applications written in TypeScript and JavaScript here:
 
-const client: WeaviateClient = await weaviate.connectToWCS(
-  'some-endpoint.weaviate.network', {
-    authCredentials: new weaviate.ApiKey('api-key'),
-  } 
-)
- 
-console.log(await client.isReady())
-```
-</details>
+- [QuoteFinder](https://github.com/weaviate/quote-finder/tree/main)
+- [NuxtVectorSearch](https://github.com/malgamves/nuxt-typescript-vector-search)
 
-Convert `sample.ts`.
+:::note v2 → v3 client documentation migration
+We are in the process of migrating our v2 client code examples to v3 as necessary. Please be patient as we work on documenting the rest.
+:::
 
-```bash
-tsc 
-```
+At the moment, we've covered the following pages. 
 
-Run the converted file.
+- [Introduction](https://weaviate.io/developers/weaviate/introduction)
+- [Quickstart](https://weaviate.io/developers/weaviate/quickstart)
+- [Starter Guides](https://weaviate.io/developers/weaviate/starter-guides)
+- [How-to: Search](https://weaviate.io/developers/weaviate/search)
+- [How-to: Manage Data](https://weaviate.io/developers/weaviate/manage-data)
 
-```bash
-node sample.js  
-```
 
-The output looks like this.
+Our [RESTful endpoints](../../api/rest/index.md) and [GraphQL functions](../../api/graphql/index.md) covered by the TypeScript client currently have JavaScript examples in the code blocks.
```

Comment:
Consider: 

The example code for the [RESTful endpoints](../../api/rest/index.md) and [GraphQL functions](../../api/graphql/index.md) that the TypeScript client supports is written in JavaScript.